helpers/getEmailBody.tsx

```diff
@@ -57,14 +67,14 @@ export default function getEmailBody({
                       padding: 20,
                       textAlign: 'center',
                     }}>
-                      <a href='https://pathology.gg'>
+                      <a href={`${game.baseUrl}`}>
                         {/* eslint-disable-next-line @next/next/no-img-element */}
-                        <img src='https://i.imgur.com/fD1SUrZ.png' alt='Pathology' />
+                        <GameLogoAndLabel id={game.id} gameId={game.id} />
```

Comment:
Haven't tested but there is a high chance this won't work, email html is very limited

Reply:
I tested! It seems to work